### PR TITLE
Readme

### LINKED ISSUE #137: README
- Write the readme, including screen shots and gifs where applicable.

### PR DESCRIPTION
**Pull request by stacyp2006**
### Participating Group Members:

- Stacy Potten
- Bret Merritt

### Code Highlights:

- Adds text, screenshots, and gifs for README

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Message/Questions for reviewer:

- PLEASE DON'T MERGE UNTIL WE PUSH UP THE GIF. 

### Issues:

- Addresses #
- Closes #137 

### Screenshots (if appropriate):

\*

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,93 @@
...
+- Babel
+- HTML5
+- CSS/SCSS
+- Travis CI
+- Heroku
+
 
 ## Set-up
 
+The deployed application can be found [HERE.](https://the-bone-yard-fe.herokuapp.com/#/)
```
Comment (nicolegooden) [inline README.md:44]: You can remove line 44 since this same line exists in the Project Description.  :)

**Review by nicolegooden — CHANGES_REQUESTED** [review]
Awesome, Stacy!  Thank you for getting this documentation set up and ready for others to view.  If you could make those few small changes I requested, that would be great.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,93 @@
-# The Bone Yard -- frontend
+# The Bone Yard -- Frontend
 
-As a dog owner, I'd like to be able to set-up "doggie play-dates" with other owners and dogs but there's no safe and functional way to do that. This app attempts to tackle that monumentous problem.
 
-The app [(which can be found here)](https://the-bone-yard-fe.herokuapp.com/#/) currently allows a user to search for dog parks in their current area, or a specific area of their choosing.
+## Project Description 
 
-## Purpose
+This application is dog park locator that allows users to search for dog parks near them or to search by city and state. Users can view details about a particular dog park, get driving directions to a specific dog park, and save dog parks they are interested in. This is a full stack application with the front-end housed on this repository. The repository for the back-end can be found [here](https://github.com/the-bone-yard/backend-repo). This project is the capstone project completed during the final module of the Front End Engineering program at the Turing School of Software and Design.
```
[the remark this answers is not in the document]
Comment (stacyp2006) [reply README.md:6]: D'oh! Thanks for catching that!

**Comment by stacyp2006** [comment]
Made those changes. Also, the icons Travis originally used for our LI/GH hyperlinks were black so they were nearly invisible in dark mode. I changed those to simple inline links.

**Review by npdarrington — APPROVED** [review]
Looks really good, thank you for putting together the Readme Stacy!

**Review by bretm9 — APPROVED** [review]
Looks great! Thanks for getting this documentation finalized 🥳

**Review by nicolegooden — APPROVED** [review]
Thank you for taking this on, Stacy!  Appreciate those little edits as well.

**Pull request merged**